front/src/common/api/osrdRailwayManagerApi.ts

```diff
@@ -0,0 +1,369 @@
+    reception_signal?: 'OPEN' | 'STOP' | 'SHORT_SLIP_STOP';
+    stop_for?: null | Interval;
+  }[];
+  speed_limit_tag?: null | string;
+  start_time: string;
+  train_name: string;
+};
+export type ConstraintDistribution = 'STANDARD' | 'MARECO';
+export type Options = {
+  use_electrical_profiles?: boolean;
+  use_speed_limits_for_simulation?: boolean;
+};
+export type Margins = {
+  boundaries: string[];
+  /** The values of the margins. Must contains one more element than the boundaries
+    Can be a percentage `X%` or a time in minutes per 100 kilometer `Xmin/100km` */
+  values: string[];
+};
+export type Items2 = {
+  /** The unique identifier of the path item.
+    This is used to reference path items in the train schedule. */
+  id: string;
+  /** The location of a path waypoint */
+  location:
+    | {
+        /** Offset in mm */
+        offset: number;
+        /** Track section identifier */
+        track: string;
+      }
+    | {
+        operational_point:
+          | {
+              /** The object id of an operational point */
+              operational_point: string;
+            }
+          | {
+              /** An optional secondary code to identify a more specific location */
+              secondary_code?: string | null;
+              /** The operational point trigram */
+              trigram: string;
+            }
+          | {
+              /** An optional secondary code to identify a more specific location */
+              secondary_code?: string | null;
+              /** The [UIC](https://en.wikipedia.org/wiki/List_of_UIC_country_codes) code of an operational point */
+              uic: number;
+            };
+        track_reference?:
+          | null
+          | (
+              | {
+                  track_id: string;
+                }
+              | {
+                  track_name: string;
+                }
+            );
+      };
+};
+export type Items3 = {
+  arrival?: null | Interval;
+  /** Position on the path of the schedule item. */
+  at: string;
+  /** State of the signal where the train is received for its stop.
+    For (important) details, see <https://osrd.fr/en/docs/reference/design-docs/timetable/#modifiable-fields>. */
+  reception_signal?: 'OPEN' | 'STOP' | 'SHORT_SLIP_STOP';
+  stop_for?: null | Interval;
+};
+export type Comfort = 'STANDARD' | 'AIR_CONDITIONING' | 'HEATING';
+export type ComponentsSchemasTransformTimetableResponsePropertiesTrainSchedulesItemsPropertiesCategoryOneOf1 =
+
+    | {
+        main_category:
+          | 'HIGH_SPEED_TRAIN'
+          | 'INTERCITY_TRAIN'
+          | 'REGIONAL_TRAIN'
+          | 'NIGHT_TRAIN'
+          | 'COMMUTER_TRAIN'
+          | 'FREIGHT_TRAIN'
+          | 'FAST_FREIGHT_TRAIN'
+          | 'TRAM_TRAIN'
+          | 'TOURISTIC_TRAIN'
+          | 'WORK_TRAIN';
+      }
+    | {
+        sub_category_code: string;
+      };
+export type TransformTimetableResponse = {
+  /** List of train schedules */
+  train_schedules: {
```

Comment:
You can reuse the type `TrainSchedule` (it's `Items`, you can rename it)
```suggestion
  train_schedules: `TrainSchedule`[]
```